### PR TITLE
Fixing active vs existing apps in event type

### LINKED ISSUE #4221: /event-types-detail - Active apps count is wrong
Currently says 2 active when 0 are active. Should say "2 apps, 0 active"

Image

### PR DESCRIPTION
**Pull request by leog**
## What does this PR do?

Fixing active vs existing apps in event type
Plus strange margin in overlay and App Card.

Fixes #4221

**Environment**: Staging(main branch) / Production

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

## How should this be tested?

Go to Event Type details page in v2 and see the correct number of active apps versus installed apps available in submenu.
Also, when opening up Giphy dialog, a strange margin appeared in the AppCard and in the overlay, both fixed, see:

<kbd></kbd>

<kbd>image</kbd>

### CODE REVIEW

**Comment by hariombalhara** [comment]
@leog There are some conflicts in it now.

**Comment by leog** [comment]
> @leog There are some conflicts in it now.

@hariombalhara Resolved.

File: apps/web/components/v2/eventtype/EventTypeSingleLayout.tsx [hunk 1]
```diff
@@ -51,6 +52,7 @@ function EventTypeSingleLayout({
   team,
   disableBorder,
   enabledAppsNumber,
+  activeAppsNumber,
```
Comment (joeauyeung) [inline apps/web/components/v2/eventtype/EventTypeSingleLayout.tsx:55]: It looks like web3 apps are not being captured by these two props

image

Comment (joeauyeung) [inline apps/web/components/v2/eventtype/EventTypeSingleLayout.tsx:55]: I believe web3 apps are not being picked up by these props

image

**Review by joeauyeung — CHANGES_REQUESTED** [review]
Tried testing with web3 apps and made a comment. Is there any other apps I should be testing with?

**Review by ciaranha — APPROVED** [review]
Looks good.

File: apps/web/components/v2/eventtype/EventAppsTab.tsx [hunk 2]
```diff
@@ -47,21 +48,25 @@ export const EventAppsTab = ({
   currency,
   eventType,
   hasGiphyIntegration,
+  giphyEnabled,
+  setGiphyEnabled,
+  paymentEnabled,
+  setPaymentEnabled,
+  rainbowEnabled,
+  setRainbowEnabled,
```
Comment (PeerRich) [inline apps/web/components/v2/eventtype/EventAppsTab.tsx:56]: I talked to @hariombalhara and this entire section needs to be extracted into packages/app-store.

each app should expose settings that can appear here.

we shouldn't create a state + component for each new app

cc @zomars for ideas

**Review by PeerRich — CHANGES_REQUESTED** [review]
https://twist.com/a/195135/ch/554218/t/3797773/

File: apps/web/components/v2/eventtype/EventAppsTab.tsx (hunk 2 above)
Comment (zomars) [reply apps/web/components/v2/eventtype/EventAppsTab.tsx:56]: Agreed, we already do some dynamic components in other areas

**Comment by zomars** [comment]
Let's focus on #4727

**Comment by leog** [comment]
Closing in favor of a more sustainable solution instead of a quick fix (initially intended for a hotfix BTW), see #4727.

**Pull request closed without merging**